aviary/models/large_turboprop_freighter/LTF.csv

```diff
@@ -0,0 +1,238 @@
+# Input Values
```

Comment:
Can you add back the header data that contains the date/time of conversion and the original filename? That info is occasionally useful. Stripping out username is ok, I do that for all converted files in the repo

Reply:
the section in create_aviary_deck that gets the time and user, never writes the comments to the output file

Reply:
I've added a fix for this to #248